### DISCUSSION TITLE
New pre-built question type | 'Fill In The Blanks'

### DISCUSSION THREAD

**Discussion by PoojaJManjooran** [Sunbird-inQuiry/Community#64] [General Discussions]
Hello team,
We are planning to develop 'Fill In The Blanks' as one of the pre-built question type in inQuiry.
@pkranga - Since your team has already worked on this enhancement, could you share any documentation you might have on the same?
@surendrasinghs @pallakartheekreddy @krgauraw @vaibhavbhuva @rajnishdargan

**Comment by pkranga** [comment]
I have already set up the discussion between Vaibhav and Amit(iGOT Team) on the UI components part. For the backend we have leveraged the same QuML specifications/API's.

We can look at reusing the components that we have built for FTB and MTF question types.
